backend/src/auth/strategy/naverAuth.strategy.ts

```diff
@@ -0,0 +1,33 @@
+import { Profile, Strategy } from 'passport-naver-v2';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable } from '@nestjs/common';
+
+require('dotenv').config();
+
+@Injectable()
+export class NaverStrategy extends PassportStrategy(Strategy) {
+  constructor() {
+    super({
+      clientID: process.env.NAVER_CLIENT_ID,
+      clientSecret: process.env.NAVER_CLIENT_SECRET,
+      callbackURL: process.env.NAVER_CALLBACK_URL,
+    });
+  }
+
+  async validate(
+    accessToken: string,
+    refreshToken: string,
+    profile: Profile,
+    done: any,
```

Comment:
```suggestion

type VerifyCallback = (error: Error, user?: any, info?: any) => void;

...
        done: VerifyCallback,
...
```
여기서 `any`를 쓰는데, 그걸 위 코드처럼 해결할 수는 있는데 어떤게 나을까요?
타입을 명시할 수 있다면 명시하는 방향이 좋겠죠??